PerfectWedding/src/main/java/com/infoshareacademy/Utils.java

```diff
@@ -221,6 +224,13 @@ public static void setVoivodeship(ServiceProvider serviceProvider) {
         serviceProvider.getLocation().setVoivodeship(listOfVoivodeships.get(Utils.scanForInt("Wybiez wojewodztwo od 1 do " + listOfVoivodeships.size(), 1, listOfVoivodeships.size(), true).get(0) - 1));
     }
 
+    public static void setTypesOfService(ServiceProvider serviceProvider) {
```

Comment:
W sumie się zastanawiam czy setTypesOfService i setVoivodeship nie lepiej gdyby było w ProvidersAdd? Nie jestem pewien czy Utils to najlepsza klasa dla tych metod.

Reply:
ale wykorzystywana jest tez przez providersEdit. ale pewnie mozemy spojrzec ogolnie na strukture klas i zawartych w niej metod, aby ustalic, co gdzie powinno sie znajdowac. na przyklad widze w Providers Add metode AskIfActive, ktora jest mniej generyczna metoday ScanForInput z Utils. Ale to jakas pozostalosc po pierwszej wersji ProvidersAdd. Jak napisalem, ogolnie mozna by sie spotkac i przegadac, strukture, choc nie wiem czy to ma sens przy przejsciu na webowa aplikacje.
Na teraz daje "GO" dla zmian Pawla :)